charts/gardener/controlplane/charts/runtime/templates/admission-controller/secret-cert.yaml

```
@@ -1,4 +1,5 @@
-{{- if .Values.global.admission.enabled }}
+{{- if and .Values.global.admission.enabled }}
```

This `and` should not be needed?
```suggestion
{{- if .Values.global.admission.enabled }}
```